src/main/kotlin/com/jaredsburrows/license/LicenseReportTask.kt

```diff
@@ -167,6 +168,12 @@ internal open class LicenseReportTask : DefaultTask() { // tasks can't be final
         val pomFile = resolvedArtifact.file
         val node = xmlParser.parse(pomFile)
 
+        // If group ID is part of ignored groups, skip this artifact
+        val groupId = node.getAt("groupId").text().trim()
```

Comment:
Add test for this?

Reply:
I'm not particularly versed in Groovy that so that'll take me some time to figure out. I'll get back to you.